internal/healthchecks/healthchecks_test.go

```diff
@@ -27,6 +28,10 @@ import (
 
 var testLogger *log.Logger = log.New(ioutil.Discard, "", 0)
 
+func expectedResultMessage(name string, result string) string {
```

Comment:
OPTIONAL: Maybe change the name to something like `generateExpectedResultMessage()`, I think it gives a better understanding of what the method does, and what you are trying to do.

Reply:
Fixed.